sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -77,7 +77,7 @@ AZ_NODISCARD az_iot_hub_client_options az_iot_hub_client_options_default();
  * @param[in] iot_hub_hostname The IoT Hub Hostname.
  * @param[in] device_id The Device ID.
  * @param[in] options A reference to an #az_iot_hub_client_options structure. If `NULL` is passed,
- * az_iot_hub_client_init() will use the default options. If using custom options, please
+ * the hub client will use the default options. If using custom options, please
  * initialize first by calling az_iot_hub_client_options_default() and then populating relevant

```

Comment:
I am guessing part of this will now line wrap to the previous line. So, auto-format?

Reply:
ah yea I clang-formatted and it didn't do anything. but if I move that line back up and format it's then different. Updating